SUSE/connect
# package: added a timer for SUSEConnect --keepalive

## How to test this

1. Build the package as described on `PACKAGE.md`.
2. Copy the package into another machine (e.g. vm).
3. Install the package.

After this, you should see that there exists `suseconnect-keepalive.service` and the `suseconnect-keepalive.timer` with the `systemctl status` command. Notice that it will be marked as `inactive` for the `suseconnect-keepalive.service` unit. This is *fine*, since this is a one-shot service that will be used by the timer. The important thing is that the timer unit is enabled and running after installing the package.

Moreover, try to force the update of this package `zypper in -f <package>`, and you should get the same result, and from the logs of the timer you should see that it got updated.

If you want to make sure that the whole thing will work, instead of waiting for 24h, you can decrease the timer to something like `OnCalendar=hourly` and `RandomizedDelaySec=10min`. This way it will only take around 1 hour instead of 24.

However you decrease it, you should see that at a certain time the `suseconnect-keepalive.service` unit will be called, and thus the command we are aiming for.
